pkg/eraser/eraser.go

```diff
@@ -209,6 +291,28 @@ func main() {
 		os.Exit(1)
 	}
 
+	// read excluded values from excluded configmap
+	data, err = os.ReadFile("/run/eraser.sh/excluded/excluded")
+	if err != nil {
+		if os.IsNotExist(err) {
+			log.Info("excluded configmap does not exist", "error: ", err)
+		} else {
+			log.Error(err, "failed to read excluded values")
+			os.Exit(1)
+		}
+	} else {
+		var result map[string][]string
```

Comment:
A struct is a little bit more type-safe, more extensible, and less error prone when we access the `Excluded` field. Then we can unmarshal into the struct.
```go
type exclusionList struct {
	Excluded []string `json:"excluded"`
}
```